2.ActiveBorrowerswithCTEs.sql

```diff
@@ -0,0 +1,11 @@
+use Tech_Library;
+WITH BorrowerCount AS (
+    SELECT BorrowerID, COUNT(*) AS BorrowedCount
+    FROM Loans
+    WHERE DateReturned IS NULL
+    GROUP BY BorrowerID
+)
+SELECT *
```

Comment:
again, be careful with select and only select target columns
this comment is applied to all files, you can return the full name here and the book name in the previous comment